### ISSUE TITLE
Bugs are not added / closed (or handled at all) when using side-tags

### ISSUE THREAD

**Issue by pvalena** [fedora-infra/bodhi#4138]
I've created an [update](https://bodhi.fedoraproject.org/updates/FEDORA-2020-4dd34860a3), using a side-tag.

The CVE bugs I've added manually for and I expected the rest, to be added by Bodhi.

Bugs, when not built in side-tag, are added based on "Resolves: rhbz#..." ([in changelog](https://koji.fedoraproject.org/koji/buildinfo?buildID=1617470), [commits](https://src.fedoraproject.org/rpms/rubygem-actionview/c/8ea90dc7bcf16d185415ed40a73ffe6c695b0332?branch=master)), which were both present for all relevant packages in the side-tag.

This did not happen, however.

**EDIT:** Bugs that I've added manually were handled correctly.

_ _ _ _
Note: the bugs are now updated to new version (some days after the update went stable; mistake on my side, shoul've closed them immediately), so I don't need to close them, but I will create new update soon, should I add those bugs manually?

**Comment by mattiaverga** [comment]
The "Resolves: rhbz#..." thing only works for automatic updates. For normal or side-tag updates the user is supposed to manually add the bugs, since they already have to manually create the update.

**Comment by pvalena** [comment]
> The "Resolves: rhbz#..." thing only works for automatic updates. For normal or side-tag updates the user is supposed to manually add the bugs, since they already have to manually create the update.

Could we then turn this into RFE for WebUI? When selecting a specific `side-tag`, not only would the builds be filled in correctly, but also the bugs?

**Comment by pvalena** [comment]
I see- this does not work for stable Fedora updates as well, then- it wouldn't be limited to side-tags, but more of a general UX enhancement.

**Comment by mattiaverga** [comment]
This would be extremely hard as it has do be done completely in javascript and I don't even know if it's possible. In backend we use Koji's python module for most of the work.

I don't know if you noticed, but after you select some builds (or a side-tag) in the webUI form, clicking on the bug field will show you the open bugs related to the packages you selected.
Isn't that enough?

**Comment by pvalena** [comment]
Yes, the menu __is__ handy, but adding it automatically would help, actually.

If I have 14 packages (or more), I have to count the number of bugs I enter, so none would be left out (there're more opened bugs than those with update). And sometimes I need to input some CVE's, additionally, and then I have to check _with the changelog of every package_ which CVE/bug was resolved. That did actually lead us to currently disable the monitoring, as the rest of the upgrade/update/testing is automated.

In case it can't be relayed to said python script via some ajax, I'd be glad to write it in (pure) javascript (granted there's an access to the relevant data).
